lib/mini_sql/abstract/prepared_binds.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+module MiniSql
+  module Abstract
+    class PreparedBinds
+
+      # For compatibility with Active Record
+      BindName = Struct.new(:name)
+
+      def bindinize(sql, *params)
```

Comment:
not sure about this name .... maybe a simple `bind` or `prepare_bind` ?

Reply:
agree, rename to `bind`